templates/event-export.php

```diff
@@ -1,65 +1,76 @@
 <?php
-	$args = Array(
-		'post_type' =>  'campaign',
-		'per_page'  =>  -1
+/**
+ * Event Export in admin
+ *
+ * Event exporting template
+ *
+ * @package WordPress
+ * @subpackage community-portal
+ * @version 1.0.0
+ * @author  Playground Inc.
+ */
+
+	$args = array(
+		'post_type' => 'campaign',
+		'per_page'  => -1,
```

Comment:
This should be changed to 'post_per_page', right now it's only pulling in 10 campaigns and 10 activities.

Reply:
Fixed.  Also fixed an issue with activities.